omero/users/cli/sessions.rst

```diff
@@ -74,7 +74,9 @@ Some of the options available to the :program:`omero login` command are:
 
 .. option:: -w PASSWORD, --password PASSWORD
 
-    Set the password to use for the connection.
+    Set the password to use for the connection. Since 5.4.1, the password can
+    be set using the :envvar:`OMERO_PASSWORD` environment variable. The variable
+    will be ignored if -w or --password is used.
```

Comment:
I think the ``-w`` and ``--password`` should be formatted, probably in double inverted ticks. Now, especially the ``-w`` is a bit drowning in the surrounding text.

Reply:
I can open a follow up PR to fix that